website/static/js/comment.js

```diff
@@ -14,6 +14,187 @@ var osfHelpers = require('js/osfHelpers');
 var CommentPane = require('js/commentpane');
 var markdown = require('js/markdown');
 
+var caret = require('Caret.js');
+var atWho = require('At.js');
```

Comment:
Do we need to require these here, even though we don't use either variable in this file?